addons/zoom-slider/addon.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Zoom slider",
+  "description": "Replaces the regular zoom buttons with a slider.",
```

It would probably make sense to also mention the number input.
```suggestion
  "description": "Replaces the code area zoom buttons with a slider and number input.",
```